app/lib/app/SecvisogramPage/View.js

```diff
@@ -332,12 +335,18 @@ function View({
     const tracking = formValues.doc.document.tracking
     let prefilledSummary = ''
     let prefilledLegacyVersion = ''
+
     if (tracking?.version && tracking.revision_history?.length) {
-      const majorVersion =
-        typeof tracking.version === 'string'
-          ? parseInt(tracking.version.split('.')[0])
-          : tracking.version
-      if (majorVersion >= 1) {
+      const semverVersion = semver.valid(tracking.version)
+      const initialPublicationVersion = semver.parse('1.0.0')
+
+      // prefill only if Integer versioning OR Semantic version is greater than 1.0.0 (after initial Publication)
+      if (
+        !semverVersion ||
```

Comment:
I'd find this a more clear naming:

```suggestion
      const isSemverVersion = semver.valid(tracking.version)
      const initialPublicationVersion = semver.parse('1.0.0')

      // prefill only if Integer versioning OR Semantic version is greater than 1.0.0 (after initial Publication)
      if (
        !isSemverVersion ||
```

Reply:
Yes and No :)

semver.valid() is null if the version is no valid semver string and a semver object representing the version if it is. In the case of a valid semver string the naming would not match.

Reply:
What about `Boolean(semver.valid("tracking.version"))` ?

Reply:
We need the semverVersion object for comparison

Reply:
:see_no_evil: indeed